src/interface/update.cpp

```diff
@@ -31,14 +29,31 @@ namespace parthenon {
 
 namespace Update {
 
-TaskStatus FluxDivergenceBlock(std::shared_ptr<MeshBlockData<Real>> &in,
-                               std::shared_ptr<MeshBlockData<Real>> &dudt_cont) {
+KOKKOS_INLINE_FUNCTION
+Real FluxDiv_(const int l, const int k, const int j, const int i, const int ndim,
+              const Coordinates_t &coords, VariableFluxPack<Real> v) {
```

Comment:
```suggestion
Real FluxDiv_(const int &l, const int &k, const int &j, const int &i, const int &ndim,
              const Coordinates_t &coords, const VariableFluxPack<Real> &v) {
```

See below. Passing `v` by reference is crucial for performance.